src/anm/animation/element.js

```diff
@@ -1211,11 +1245,12 @@ Element.prototype.removeModifier = function(modifier) {
     if (!this.__modifiers_hash[modifier.id]) throw new AnimationError('Modifier wasn\'t applied to this element');
     if (!modifier.__applied_to || !modifier.__applied_to[this.id]) throw new AnimationError(Errors.A.MODIFIER_NOT_ATTACHED);
     //if (this.__modifying) throw new AnimErr("Can't remove modifiers while modifying");
-    delete this.__modifiers_hash[modifier.id];
-    delete this.$modifiers[modifier.type].splice(modifier.__applied_to[this.id] - 1, 1); // delete by index
-    delete modifier.__applied_to[this.id];
+    utils.removeElement(this.__modifiers_hash, modifier.id);
```

Comment:
`removeElement` name correlates with DOMElement or Animatron Element, I thinks it's more like `disposeValue` or something.